impeller/entity/contents/filters/filter_contents.h

```diff
@@ -132,17 +132,23 @@ class FilterContents : public Contents {
   // |Contents|
   const FilterContents* AsFilter() const override;
 
-  /// @brief Determines the coverage of source pixels that will be needed
-  ///        to apply this filter under the given transform and produce
-  ///        results anywhere within the indicated coverage limit.
+  /// @brief  Determines the coverage of source pixels that will be needed
+  ///         to produce results for the indicated output limit. This is a
+  ///         reverse bounds calculation computing a source coverage from
+  ///         an intended output coverage.
```

Comment:
So, it's the inverse of `GetCoverage`?

Reply:
Correct. Is that confusing from what I wrote? Or are you suggesting a simplification?

Reply:
Oops, I lost the mention of the transform. Minor update coming...

Reply:
How about:

```
  /// @brief  Determines the coverage of source pixels that will be needed
  ///         to produce results for the indicated output coverage under the
  ///         indicated effect transform. This is essentially a reverse of
  ///         the |GetCoverage| method computing a source coverage from
  ///         an intended output coverage.
```

(I'm also renaming the `output_limit` parameter to `output_coverage` to match that text)

Reply:
Ugh, renaming the parameter and propagating it to all of the downstream methods would touch a bunch of files unnecessarily, so I'll change the comment to match instead.

Reply:
It was just a genuine question.  That doesn't match my understanding after having looked at `DirectionalGaussianBlurFilterContents`.

Reply:
Perhaps you are confused by the non-intuitive fact that blur filters expand in both directions. Just as a given edge pixel can be the input for a fringe around the source and so it expands when you apply the filter, each pixel takes input from a range of pixels around it so you need more source pixels than output pixels in order to compute the values correctly.

So, the gaussian filter **is** computing a reverse bounds by _expanding_ it...